lib/reunion.rb

```diff
@@ -0,0 +1,43 @@
+class Reunion
+
+  attr_reader :name,
+              :activities
+
+  def initialize(name)
+    @name = name
+    @activities = []
+  end
+
+  def add_activity(activity)
+    @activities << activity
+  end
+
+  def total_cost
+    @activities.map do |act|
+        act.total_cost
+    end.flatten.sum
```

Comment:
I don't think you need to flatten this map, I don't think that it will ever return a nested array.